ban/core/models.py

```diff
@@ -48,15 +48,15 @@ class Municipality(NamedModel):
     resource_fields = ['name', 'alias', 'insee', 'siren', 'postcodes']
     exclude_for_version = ['postcodes']
 
-    insee = db.CharField(length=5, unique=True)
-    siren = db.CharField(max_length=9, unique=True, null=True)
+    insee = db.CharField(length=5, unique=True, format='[\dAB]{2}\d{3}')
+    siren = db.CharField(length=9, format='\d*', unique=True, null=True)
 
 
 class PostCode(NamedModel):
     resource_fields = ['code', 'name', 'alias', 'complement', 'municipality']
 
     complement = db.CharField(max_length=38, null=True)
-    code = db.PostCodeField(index=True)
+    code = db.CharField(index=True, format='\d*', length=5)
```

Comment:
Meh, you want me to add back the PostCodeField? :p
I agree it's confusing, but the situation is that at the end only La Poste clients will be able to edit PostCode, and I guess they know about that :)
So I'm tempted to resist, and keep the regex code generic (needing a dedicated error message means a custom field back).
What d'you say? :)